pubsub/streaming_pull_test.go

```diff
@@ -513,7 +513,8 @@ func newMock(t *testing.T) (*Client, *mockServer) {
 }
 
 // pullN calls sub.Receive until at least n messages are received.
-func pullN(ctx context.Context, sub *Subscription, n int, f func(context.Context, *Message)) ([]*Message, error) {
+// Wait a provided number of seconds before cancelling.
+func pullN(ctx context.Context, sub *Subscription, n, wait int, f func(context.Context, *Message)) ([]*Message, error) {
```

Comment:
the wait as int seems like an odd choice (wrong type, no indication of the base interval).  Maybe just make it time.Duration?